src/migrations/Version20241111184121.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20241111184121 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
```

Comment:
Ideally, you should "name" your migrations so you know what's included when you return to the code after some time